### PR TITLE
xy_12

### LINKED ISSUE #176: xy_12
Description: highlighting a section of a line using two different methods

NCL Script: https://www.ncl.ucar.edu/Applications/Scripts/xy_12.ncl
NCL Plots: https://www.ncl.ucar.edu/Applications/Images/xy_12_1_lg.png
https://www.ncl.ucar.edu/Applications/Images/xy_12_2_lg.png

Category: XY

### PR DESCRIPTION
**Pull request by hCraker**
Closes #176 
image
image

### CODE REVIEW

File: Plots/XY/NCL_xy_12.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/xy_12.ncl
+    - Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/xy_12_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/xy_12_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+import math
```
Comment (michaelavs) [inline Plots/XY/NCL_xy_12.py:18]:
```suggestion

```
Imported and unused. Other than that, Great job!

File: Plots/XY/NCL_xy_12.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+##############################################################################
+# Method 1: Splitting the line into parts and coloring them differently
+U = ds.isel(time=0, lon=5, drop=True).U
+
+plt.figure(figsize=(7, 7))
+ax = plt.axes()
+
+bins = [0, 5, 20]
+# Slicing data is exclusive for the last value, to work around this we
```
Comment (clairefio) [inline Plots/XY/NCL_xy_12.py:37]: This comment is a little confusing, I would clarify what you're working around.

File: Plots/XY/NCL_xy_12.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+##############################################################################
+# Method 2: Drawing a polygon around the section of interest
+U = ds.isel(time=0, lon=84, drop=True).U
+
+plt.figure(figsize=(7, 7))
+ax = plt.axes()
+
+bins = [5, 24]
+# Slicing data is exclusive for the last value, to work around this we
```
Comment (clairefio) [inline Plots/XY/NCL_xy_12.py:72]: Same as above.

File: Plots/XY/NCL_xy_12.py (hunk 2 above)
Comment (hCraker) [reply Plots/XY/NCL_xy_12.py]: Is it less confusing now that I have updated it?

**Pull request merged**